.pre-commit-config.yaml

```diff
@@ -8,11 +8,13 @@ repos:
   - id: end-of-file-fixer
   - id: check-yaml
   - id: check-added-large-files
+
 - repo: https://github.com/psf/black
-  rev: 21.9b0
+  rev: 21.12b0
   hooks:
-  - id: black
+  - id: black-jupyter
```

Comment:
If we switch to this hook, will this still run black as it normally would at commit time?

Reply:
Yep! But with the added bonus that it can check Jupyter notebooks as well.